### ISSUE TITLE
Add random seed tracking for reproducibility

### ISSUE THREAD

**Issue by noahgolmant** [noahgolmant/skeletor#10]
Add random seed as an argument

Add function to set the seed for various libraries (random, numpy, pytorch, tensorflow, etc.)

Add test to see that it's actually reproducible for all of these
